docs/FLYSKY_FRM303.md

```diff
@@ -0,0 +1,112 @@
+# mLRS Documentation: FlySky FRM303 Transmitter Module #
+
+([back to main page](../README.md))
+
+The FlySky FRM303 transmitter module is a nearly perfect option for building mLRS equipment in the 2.4 GHz frequency range. Its main drawback is its price. It also has a small MCU which limits the available mLRS features. However, no hardware tweaking is required (although mods like the OLED mod are possible).
+
+<table>
+  <tbody>
+    <tr>
+      <td>Frequency Band</td>
+      <td>2.4 GHz</td>
+    </tr>
+    <tr>
+      <td>Max. RF Output Power</td>
+      <td>30 dBm (1 W) (the hardware is capable of 33 dBm/2 W, which is not enabled by default however)</td>
+    </tr>
+    <tr>
+      <td>Supported Modes</td>
+      <td>50 Hz, 31 Hz, 19 Hz</td>
+    </tr>
+    <tr>
+      <td>Weight</td>
+      <td>65 grams without antenna</td>
+    </tr>
+  </tbody>
+</table>
+
+**Important: Never power up without an antenna connected! Otherwise you risk damaging the RF section.**
+
+- [Product Page](https://www.flysky-cn.com/frm303description)
+
+## As Tx Module ##
+
+The description is for the default firmware version ('USB' version, indicated by a '-usb' label in the firmware file name).
+
+The FRM303 provides two connectors, a 5-pin JST-GH connector at the bottom edge and a USB-C connector at top edge, and a 5-way button. Their usage is as follows:
+
+JST-GH connector (from left to right when viewd from top):
+
+- serial Rx
+- serial Tx
+- VCC
+- GND
+- in port
+
+USB-C connector:
+
+- com/cli port
+
+5-way button:
+
+- bind button (to enter bind mode, hold down the button for ca 4 seconds)
+
+Notes:
+
+- For this 5-way button, 'hold down' means to move it to the bottom, not to press it down
+- To enter the system bootloader for flashing, hold down the button during power up (or use the CLI, Lua script, or OLED if available)
```

Comment:
Perhaps just one line:

To enter the system bootloader for flashing, hold the button in the down direction during power up (or use the CLI, Lua script, or OLED if available)

Reply:
great!